packages/app/components/navbar/MobileMenu.tsx

```diff
@@ -11,7 +11,7 @@ export default function MobileMenu() {
   const toggle = () => setIsOpen(!isOpen);
 
   return (
-    <div className="flex items-center justify-end w-full gap-4 md:hidden">
+    <div className="z-10 flex items-center justify-end w-full gap-4 md:hidden">
```

Comment:
Why this z index?
This is happening with this change. What should we do here? We can increase the overlay z index if needed

![Screenshot 2023-08-25 at 14 21 48](https://github.com/SwaprHQ/stackly-ui/assets/23079677/4746d83f-8a25-4c92-ad39-4da345a0eb31)

Reply:
<img width="377" alt="image" src="https://github.com/SwaprHQ/stackly-ui/assets/5664434/3c78d3d4-93da-4eb6-b96f-e9e234a90800">

The idea was to fix the arrow from overlapping the menu. Will try to find a solution.

Reply:
fixed [c5423a6](https://github.com/SwaprHQ/stackly-ui/pull/65/commits/c5423a6ecb804f003b7cb41d7f36c5da3c0acafb)